bills-app/src/main/webapp/script.js

```diff
@@ -767,26 +767,35 @@ const mockResponse = {
     }]
 }
 
-// Fetched the upcoming bills from the api
-const getUpcomingBills = () => {
+function getUpcomingBills() {
     //api key is declared in api.js as: 'proPublicaApiKey'
-    const url = "https://api.propublica.org/congress/v1/bills/upcoming/house.json";
+    const url="https://api.propublica.org/congress/v1/bills/upcoming/house.json";
```

Comment:
```suggestion
    const url = "https://api.propublica.org/congress/v1/bills/upcoming/house.json";
```
You can keep spacing around the = in javascript :)